easy-deploy-script/easy-deploy-viral-ngs.sh

```diff
@@ -226,7 +240,12 @@ function install_viral_ngs_conda(){
 
 function install_viral_ngs_git(){
     if [ ! -L "$VIRAL_NGS_PATH" ]; then
-        git clone https://github.com/broadinstitute/viral-ngs.git "$VIRAL_NGS_PATH"
+        # First optional argument specifies non-master branch
+        if [[ $# -eq 2 ]]; then
+            git clone https://github.com/broadinstitute/viral-ngs.git --branch $2 "$VIRAL_NGS_PATH"
```

Comment:
Should this be `$1` for the first arg since this is a function, and then the call below would be `install_viral_ngs_git $2` to pass the command line arg with the branch name?

Reply:
Done